app/controllers/admin_controller.rb

```diff
@@ -6,7 +6,7 @@ class AdminController < ApplicationController
 
   # For now, admin controllers allow anyone who is logged in
   def authorize_access
-    unless current_user.present?
+    unless can? :see_admin_pages
```

Comment:
Right, see, like this! So I don't think we need the `can_see_admin_pages?` helper method (that isn't being used here anyway)

Reply:
Yup.